mmdt/mmdt/settings.py

```diff
@@ -1,38 +1,15 @@
-"""
-Django settings for mmdt project.
-
-Generated by 'django-admin startproject' using Django 4.2.4.
-
-For more information on this file, see
-https://docs.djangoproject.com/en/4.2/topics/settings/
-
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/4.2/ref/settings/
-"""
-
```

Comment:
It appears that you have removed comment lines and whitespace. Please ensure that these changes are not updated. If you wish to make formatting-related changes, kindly create a separate PR for them. Consolidating only the necessary changes in one PR helps maintain a clear and singular focus for each PR.